app/javascript/components/Editor.tsx

```diff
@@ -378,7 +377,9 @@ export function Editor({
                   <ResultsTab />
                 </div>
                 <InstructionsPanel {...panels.instructions} />
-                {panels.tests ? <TestsPanel {...panels.tests} /> : null}
+                {panels.tests ? (
+                  <TestsPanel language={track.slug} {...panels.tests} />
```

Comment:
@kntsoriano The `track.highlightjs_language` needs to be used for anything with highlightjs, not the `slug`. I've fixed this.